client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -73,7 +74,24 @@ def class_module_names(model_class: Type[VertaModelBase]) -> Set[str]:
     return modules_found
 
 
-# TODO VRD-682 convert module names to pkg names here
 def package_names(module_names: Set[str]) -> Set[str]:
-    """Return a set of all packages detected in the provided set of base module names."""
-    return module_names
+    """Return a set of distribution package names for all 3rd-party import
+    modules in the provided set.
+
+    Package distributions don't exist for modules in the Python standard library.
+    This fact is used to capture only 3rd-party packages.  Modules associated
+    with a single package name are easily captured.  Modules associated with a
+    namespace package (e.g.`google`), may come from a variety of distribution
+    packages.  To avoid making an arbitrary choice in those cases, the main
+    namespace package name is fetched from the distribution metadata.
+    """
+    pkg_dist = packages_distributions()
+    pkgs = set()
+    for mod in module_names:
+        dist_pkgs: Union[List[str], None] = pkg_dist.get(mod)
```

Comment:
The docstring makes it sounds like `dist_pkgs` is only `None` if `mod` comes from the standard library, but this doesn't seem to be the case:

![Screen Shot 2023-03-08 at 1 42 00 PM](https://user-images.githubusercontent.com/96442646/223856878-26d884fe-61ed-4493-bee0-67cf01158e64.png)

We might want to at least log those cases in an `else` clause below.

Reply:
This seems like something we'll need to investigate, but I propose we do it in a separate PR.  I have a new branch with the fix for modules introduced as Class instances (problem you raised in this [slack thread](https://vertaai.slack.com/archives/C04NVN48LAV/p1678310608042619)). We could wrap the two things together in that

Reply:
A follow-up PR is alright, but please don't bundle it together with the other issue—they're unrelated things, as far as I can tell.